doc/snipe.txt

```diff
@@ -75,6 +83,29 @@ terminal mode respectively):
     let g:snipe_highlight_gui_color      = '#dfffaf'
     let g:snipe_highlight_cterm256_color = 'blue'
     let g:snipe_highlight_cterm_color    = 'red'
+
```

Comment:
As I mentioned in the other comment, do we need to allow users to opt in to this behavior? Your suggested approach is simpler and more intuitive.

Reply:
Yeah that makes sense. I just didn't know if you really wanted that feature. I will clean the code up.